commands/command_unlock.go

```diff
@@ -31,11 +33,16 @@ func unlockCommand(cmd *cobra.Command, args []string) {
 			Exit("Unable to determine path: %v", err.Error())
 		}
 
+		unlockCheckFileStatus(path)
```

Comment:
I'm not a huge fan of the absence of a return value here. That's not to say that all functions need to return something, but the fact this function `Exit()`s from within forces the reader to inspect the body of the function to know that the calling code may not progress beyond this point.

I was originally going to suggest:

```go
if err := unlockCheckFileStatus(...); err != nil {
         // err -> nil if file unmodified, or ErrFileIsModified
}
```

but that turned into:

```go
if m, err := unlockCheckFileStatus(...); err != nil {
         // handle error
} else {
        if m { Exit(...) }
        else { Error(...) }
}
```

to handle differentiating between an error in getting the modified status of a file, and whether or not the file is actually modified. One way you can do that is `*bool`, but I think the greater truth that this function is unnecessary.

I'd suggest either:

- Removing the function and promoting it's body inline
- Adding a comment suggesting that calling this function _can_ bail out.